src/chrome/content/rules/SlovakGovernment.xml

```diff
@@ -1,15 +1,17 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://mosr.sk/ => https://www.mosr.sk/: (28, 'Connection timed out after 20001 milliseconds')
-Fetch error: http://www.mosr.sk/ => https://www.mosr.sk/: (28, 'Connection timed out after 20001 milliseconds')
-Fetch error: http://justice.gov.sk/ => https://www.justice.gov.sk/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-Fetch error: http://www.justice.gov.sk/ => https://www.justice.gov.sk/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-Fetch error: http://justice.sk/ => https://www.justice.gov.sk/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-Fetch error: http://www.justice.sk/ => https://www.justice.gov.sk/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-
+    Nonfunctional hosts:
+
+    certificate mismatch:
```

Comment:
They have only cert chain issues.

Reply:
Please just change the comment to pass our tests. We can't mitigate cert chain issues.